src/mlpack/core/util/mlpack_main.hpp

```diff
@@ -230,6 +230,8 @@ PARAM_FLAG("copy_all_inputs", "If specified, all input parameters will be deep"
     " copied before the method is run.  This is useful for debugging problems "
     "where the input parameters are being modified by the algorithm, but can "
     "slow down the code.", "");
+PARAM_FLAG("no_sanity_checks", "If specified, the input matrix is checked for"
```

Comment:
Why `no_sanity_checks`?  In this case, if we specify the flag, then we *are* doing the sanity checks.  I might suggest a name like `check_input_matrices` or something like that.

Reply:
You are right, I have corrected it now